### ISSUE TITLE
Metabox Extension

### ISSUE THREAD

**Issue by manikbajaj** [reduxframework/redux-extensions#241]
Dear Team,

I have purchased and downloaded the extension.
I have uploaded this extension to my redux extension and setup the config and loader properly (tested with different extension and found works well)
After followed the documentation about the metabox given on docs. Nothing happened. 

Please advise!!

Thanks

**Comment by dovy** [comment]
We don't really use the extension loader now, we built it into the Redux API. Use the builder http://build.reduxframework.com and then follow the instructions in there. Also download a new version of Metaboxes and follow the config. Just update your opt_name.  :)

**Comment by manikbajaj** [comment]
Hi Dovy
The problem is that I am not being able to login into my Builder account using the same credentials through which I have bought the extensions. Please have a look at the snapshot. I've tried my best but in vain. 

Regards 
Manik 
![redux_builder_ _log_in](https://cloud.githubusercontent.com/assets/3671946/11531931/6267e46e-9926-11e5-9d29-88c49c0f9f93.jpg)

**Comment by manikbajaj** [comment]
Hi Dovy,

I have finally able to login. I didn't found any metabox option. It was same as per my free account. 

Please let me know how i can use meta box extension It is really important and i am got stuck into this,

thanks

**Comment by ericwallen** [comment]
They do not support metaboxes at all. Stole $200 of my money.

Kevin is a horrible person.

**Comment by ericwallen** [comment]
Treated like dirt after spending $200 with Redux

I agree witht the first review and am not suprised to see it. Kevin at redux is one of the rudest people I have ever met. 

After asking simple questions I recieved a mountain of attitude and no answers. I had purchased the metabox extention for $125 and was not able to install it. Keving insisted I purchase premium support for another $75 before he would help me so I did.

$200 later and being talked to like a peice of garbage they try to blame me even though they will not supply a link to documentation that works.

4k worth of WP assets and 10 years later I have never been treated by anyone in the WP so badly as I have by Redux. 

I highly reccomend not to deal with these guys.

Steeling my money is one thing but the anger comes from the way these guys treat people. After all the stuff I deal with in life I don't need a chump like Kevin talking down to me as if I am not even human.

DO NOT USE REDUX
